buildout.cfg

```diff
@@ -1,13 +1,14 @@
 [buildout]
 develop = .
-parts = interpreter test
+parts = interpreter test 
 
 [interpreter]
 recipe = zc.recipe.egg
 eggs = zope.globalrequest
-interpreter = python
+interpreter = zopepy
```

Comment:
well, when you  create a ``virtualenv  .`` (as many developer do) then ``bin/python`` is already the virtualenv one and so buildout fails to override a used python file.